locales/US_en.json

```diff
@@ -1,23 +1,60 @@
-{
-	"nav": {
-		"brand": "Pretendo"
-	},
-	"home": {
-		"header": {
-			"title": "Pretendo",
-			"text": "An open source Nintendo Network replacement service"
-		}
-	},
-	"about": {
-		"caption": "What is this?",
-		"title": "About",
-		"text": "Pretendo is an open source Nintendo Network replacement that aims to build custom servers for the WiiU and 3DS family of consoles. Our goal is to preserve the online functionality of these consoles, to allow players to continue to play their favorite WiiU and 3DS games to their fullest capacity."
-	},
-	"credits": {
-		"caption": "The people responsible",
-		"title": "Credits"
-	},
-	"footer": {
-		"text": "Copyright 2020 PretendoNetwork - Site designed and developed by mrjvs, ben and jip"
-	}
+{
+	"nav": {
+		"brand": "Pretendo"
+	},
+	"home": {
+		"header": {
+			"title": "Pretendo",
+			"text": "An open source Nintendo Network replacement service"
+		}
+	},
+	"about": {
+		"caption": "What is this?",
+		"title": "About",
+		"text": "Pretendo is an open source Nintendo Network replacement that aims to build custom servers for the WiiU and 3DS family of consoles. Our goal is to preserve the online functionality of these consoles, to allow players to continue to play their favorite WiiU and 3DS games to their fullest capacity."
+	},
+	"credits": {
+		"caption": "The people responsible",
+		"title": "Credits",
+		"redducks": "Project owner and lead developer",
+		"caramelkat": "Miiverse research and development",
+		"superwhiskers": "crunch library development",
+		"supermariodabom": "Console and other system research",
+		"rambo6glaz": "Network installer and console research",
+		"mrjvs": "Web development and early eShop research",
+		"kxpler442b": "Web development",
+		"jipfr": "Web development",
+		"kinnay": "None of this would be possible without the research and help done by Kinnay on PRUDP and nex"
+
+	},
+	"footer": {
+		"text": "Copyright 2020 PretendoNetwork - Site designed and developed by mrjvs, ben and jip"
+	},
+	"connecting": {
+		"title": "Connecting",
+		"text": "Pretendo is currently not usable by the general public. When the network becomes public, this will contain steps on connecting to the network"
+	},
+	"cemu": {
+		"title": "Cemu",
+		"text": "Pretendo is currently not usable by the general public. When the network becomes public, this will contain steps on connecting to the network using Cemu"
+	},
+	"faq": {

```

Comment:
Maybe make an array of questions instead of assigning aN values?

Reply:
Do you mean this?

```
{ "faq": [ 
    { "question" : "Here I write the first question", "answer" : "And here I write the first answer" } , 
    { "question" : "Same thing but it's the first question", "answer" : "Again, same thing but I write the second answer" } 
] }
```

Do I have to change anything in the FAQ page?
BTW thanks for the help!

Reply:
I’d probably add a questions field to faq so that you can still add the translations for the general title.

However, maybe wait with this a bit. I’ve asked the  other web people what they think (array or your current setup?) and I’ll let you know what they say

Reply:
Ok. Thank you very much!

Reply:
Or did you mean something like this?

```
{
all the other stuff

          "faq": {
                      "questions":[ "Question 1", "Question 2", "Question 3" ],
                      "answers":[ "Answer 1", "Answer 2", "Answer 3" ]
}
}
```
and in the faq page I use `locale.faq.questions[0]` when I want to insert question 1, and `locale.faq.answers[0]` to insert answer 1.
I'm sorry for my dumb questions, I'm fairly new to JSON, and javascript in general.
Again, thanks for your help!

Reply:
I was thinking ```JSON
"faq": {
  "title": "FAQ",
  "questions": [
    {
      "question": "What's 9+10?",
      "answer": "21"
    },
    {
      "question": "What's 9+10?",
      "answer": "21"
    }
  ]
}``` and then forEach it. However, beforer making such a decision I think the other web people should get involved

Reply:
Ok. Thanks!

Reply:
I agree with @JipFr on this. Maybe this could be applied to the credits as well? It feels a bit odd using the developer name as the key. Using an array and looping would allow us to add or remove a credit just in the translation file and not have to update the actual markup

Reply:
Hi, everyone. Thanks for all your support. So... I will put the FAQ and the credits in an array.
BTW for the credits would this be ok?

```
"credits": {
	"caption": "The people responsible",
	"title": "Credits",
	"people": [
		{
		"name": "Jonathan Barrow (RedDucks)",
		"caption": "Project owner and lead developer"
		"link": "https://github.com/RedDuckss"
		"picture": "https://github.com/redduckss.png"
		},
		etc.
		]
```

But I'm afraid I won't be able to edit the home.hbs and the faq.hbs, because... I don't know how to code. I searched on the internet for a guide, but I'm lost. I'm sorry. 😓

I will be happy to put the FAQ and the credits in an array, but I can't do more than that.

Reply:
https://handlebarsjs.com/guide/builtin-helpers.html#each this should help you with that ;)

Reply:
Thanks!